src/platform_impl/web/stdweb/canvas.rs

```diff
@@ -122,7 +122,11 @@ impl Canvas {
         F: 'static + FnMut(ScanCode, Option<VirtualKeyCode>, ModifiersState),
     {
         self.on_keyboard_release = Some(self.add_user_event(move |event: KeyUpEvent| {
-            event.prevent_default();
+            let event_key = &event.key();
+            let is_key_string = event_key.len() == 1 || !event_key.is_ascii();
+            if !is_key_string {
+                event.prevent_default();
+            }
```

Comment:
Shouldn't this be put inside `on_keyboard_press` instead of `on_keyboard_release`?

Reply:
Ouch, yes, of course. Thanks for spotting!